quarkus-bon-jova-rockstar/deployment/src/main/resources/dev-ui/qwc-bon-jova-rockstar-endpoints.js

```diff
@@ -23,7 +25,7 @@ export class QwcBonJovaRockstarEndpoints extends LitElement {
             color: var(--lumo-primary-text-color);
             text-decoration: none;
         }
-        
+
         .cardGrid {
             display: flex;
             flex-wrap: wrap;
```

Comment:
I would also remove `flex-wrap: wrap` and `align-items: stretch`, and add `justify-content: space-between;`. Then make the `qui-card` that contains the codeblock `width: 100%`;. That will fill the space (when you expand the grid) with the code using as much as possible.
![Screenshot_20240503_103334](https://github.com/quarkiverse/quarkus-bon-jova-rockstar/assets/6836179/167d80d1-cc98-4382-9c83-76457a667216)

Reply:
Well, this is odd. I've applied your suggestions, and I've still got the same console error, and blank screen. But it looks like you can't reproduce, based on the screencap above?

<img width="1694" alt="image" src="https://github.com/quarkiverse/quarkus-bon-jova-rockstar/assets/11509290/2889943f-78e3-4500-8fbc-f3510817b338">